web/themes/custom/pineapple/templates/content/node--recipe--full.html.twig

```diff
@@ -102,7 +103,7 @@
       </div>
     </div> <!-- End of recipe-body -->
     {% set author_node = content.field_author|field_value|first['#node'] %}
-    <aside class="recipe-sidebar">
+    <aside {{ attributes.addClass('recipe-sidebar').setAttribute('data-card', '') }}>
```

Comment:
You don't/should not need to do this, you can just add the data attribute normally. Attributes is also a reserved variable for the node

Reply:
done